### DISCUSSION TITLE
Remote Arming and Disarming alarm

### DISCUSSION THREAD

**Discussion by sensormatic1985** [taligentx/dscKeybusInterface#226] [dscKeybusInterface Discussion]
good night I have my alarm panel linked to the home assistan platform how can I arm it remotely arm it and disarm it remotely from home assistant

**Comment by kricon** [comment]
Do you have NPN transistor connected as per "Virtual keyboard" wiring instructions? Transistor connected is required to be able of writting onto keybus (arming/disarming etc)
